### ISSUE TITLE
Change google.com to wikipedia.org

### ISSUE THREAD

**Issue by aledovsky** [alex/what-happens-when#130]
While Google's front-page is undoubtedly the single most known site on the Internet, authoritative details of underlying back-end operations that result in HTML page brought to end-user's device are either unknown, scarce, outdated or non-verifiable. For the sake of _"No[t] skipping out on anything"_, it is a must that we describe processes taking place both on client **and** server side as detailed as possible. 

One can't argue that Wikipedia is also a _[very widely](http://www.alexa.com/siteinfo/wikipedia.org)_  recognized website, however best thing about it in comparison with Google is that it is a non-profit organization and runs its operations on a [known OSS stack](https://meta.wikimedia.org/wiki/Servers). I would also expect that inquiring Wikimedia sysops/admins about some additional specifics would be more successful than trying a same feat with Google. 

@alex What's your take on this?

**Comment by kojiromike** [comment]
:+1:

**Comment by aledovsky** [comment]
@kojiromike Judging by [this ticket](https://phabricator.wikimedia.org/T35890) ~~SPDY~~ HTTP/2.0 support is in development pipeline for Wikipedia. On HSTS, well, last time I checked `google.com` also doesn't set it. So, it makes no difference - either way we would desribe it from _what if_ point of view.

**Comment by gitcnd** [comment]
Yes - we absolutely need to avoid using "google.com" - because (eg: see my DHCP bug) - this happens to be the default landing domain for search queries as well, which makes the distinction between "direct" and "search" really confusing to explain.
(For example - if you use DHCP, and wake up your laptop and really quickly put "wikipedia.org" [enter] in there, you'll end up at: https://www.google.com/search?q=wikipedia.org )

I also like the ",org" ending - much better than .com for explaining stuff.

Wikipedia is not a good substitute though: it's not secure, and HSTS is really important.

**Comment by kojiromike** [comment]
What about example.org?

**Comment by aledovsky** [comment]
Well, the IANA test domains are pretty good candidates, and I at first considered them to replace `google.com` but their _ops side_ is not much known [the main reason to change from Google, remember]. Running simple `curl -v example.org` also brings some questions:
- What is the webserver software? `Server: ECS (ewr/1584)` is [pretty much unheard of](https://stackoverflow.com/a/22499836)
- How is caching designed? `X-Cache: HIT`/`Etag: "359670651"`
- Custom headers? `x-ec-custom-error: 1`

Also, there are no Cookies/HTTPS/Javascript/geo-DNS and list goes on...

**Comment by unwiredbrain** [comment]
How about:

- the [Electronic Frontier Foundation](https://www.eff.org/) (EFF)
- the [OpenSSL](https://www.openssl.org/) project
- the [Internet Engineering Task Force](https://www.ietf.org/) (IETF)
- the [Linux Kernel Archives](https://www.kernel.org/)

**Comment by aledovsky** [comment]
@unwiredbrain Any site would do (even `google.com`) if there's publicly disclosed information available regarding _ops_.

**Comment by BenjaminProgram** [comment]
We could include both under two separate files.

**Comment by uschwar** [comment]
Absolutely!
And once it's done, it should also be published on wikipedia.org. Actually, why not put it there straight away?

**Comment by StevenBlack** [comment]
@uschwar the Wikipedia deletionists won't allow this.  Lack or [references](http://en.wikipedia.org/w/index.php?title=Wikipedia:Verifiability&redirect=no#Reliable_sources) is one reason, and others like [no original research](http://en.wikipedia.org/wiki/Wikipedia:No_original_research).

What I'm saying is, when it comes to posting to Wikipedia, nothing is simple.

**Comment by kojiromike** [comment]
Nothing is simple,
Except the above statement.
Well, it seemed simple.

**Comment by BenjaminProgram** [comment]
@StevenBlack I am very involved in Wikipedia and we could easily fix those issues but the largest issue would be content that is not wikified or is Unencyclopedic.

**Comment by PiRSquared17** [comment]
The current format is unencyclopedic. You could try to get it on Wikiversity though, if it qualifies as a "learning resource" (I think it does).

**Comment by TheInitializer** [comment]
Lol. When I type "g" into my URL bar github.com pops up...

I kinda agree with this.
